src/main/java/com/google/devtools/build/lib/actions/RemoteFileStatus.java

```diff
@@ -0,0 +1,8 @@
+package com.google.devtools.build.lib.actions;
+
+import com.google.devtools.build.lib.actions.FileArtifactValue.RemoteFileArtifactValue;
+import com.google.devtools.build.lib.vfs.FileStatusWithDigest;
+
+public interface RemoteFileStatus extends FileStatusWithDigest {
```

Comment:
Add a comment explaining when it's appropriate to use this.

Reply:
Done.